crates/driver/src/infra/config/file/mod.rs

```diff
@@ -293,6 +293,24 @@ struct LiquidityConfig {
     /// Liquidity provided by a Balancer V2 compatible contract.
     #[serde(default)]
     balancer_v2: Vec<BalancerV2Config>,
+
+    /// The base URL used to connect to subgraph clients.
+    graph_api_base_url: Url,
```

Comment:
nit: maybe slightly nice to use `Option<Url>`, which implements `Default` and then `unwrap_or` with out constant when we convert it to the liquidity specific config object.

Reply:
done